contrib/win32/openssh/OpenSSHTestHelper.psm1

```diff
@@ -83,6 +83,9 @@ function Set-OpenSSHTestEnvironment
         "PostmortemDebugging" = $Script:PostmortemDebugging
         "NoLibreSSL" = $Script:NoLibreSSL
         }
+
+    #start service
+    Start-Service -Name sshd
```

Comment:
I think we can remove line 169 (here is the block)
    if(-not (Test-Path $OpenSSHConfigPath -pathType Container))
    {
        #starting the service will create ssh config folder
        start-service sshd
    }